tlsr9/CMakeLists.txt

```diff
@@ -30,13 +32,20 @@ zephyr_include_directories(drivers/${SOC})
 
 # add for B92 headers
 zephyr_include_directories(drivers/${SOC}/ext_driver)
+if (NOT CONFIG_SOC_RISCV_TELINK_TL321X)
 zephyr_include_directories(drivers/${SOC}/ext_driver/driver_lib)
+else()
+zephyr_include_directories(drivers/${SOC}/ext_driver/driver_internal)
+endif()
 zephyr_include_directories(drivers/${SOC}/ext_peripherals/codec_0581)
 zephyr_include_directories(drivers/${SOC}/flash)
 
 # soc.c reference sources
+if (NOT CONFIG_SOC_RISCV_TELINK_TL321X)
 zephyr_library_sources(drivers/${SOC}/clock.c)
 zephyr_library_sources(drivers/${SOC}/analog.c)
+zephyr_library_sources(drivers/${SOC}/stimer.c)
```

Comment:
I check with this here (3a3bc5b991d0d8b25b9199eaae212851e5cc3dcc).
We might add the stimer.c for the old file structure of the last TL321x driver version. 
We should remove it because this file has been moved into our BLE lib in the new file structure.